git_greppers/github_handler.sh

```
@@ -0,0 +1,120 @@
+
+# Function to find PR information for a commit
+find_github_pr_info() {
+  local commit_hash=$1
+  local repo_identifier=$2
+  local pr_mr_info="None found"
+  local pr_mr_number=""
+  
+  debug 2 "Looking for GitHub PR info for commit: $commit_hash" >&2
+  
+  # Check if the commit message contains PR references
+  local commit_msg=$(git show -s --format=%B "$commit_hash")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)